Traceback (most recent call last):
  File "/home/trade/anaconda3/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/trade/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/trade/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/trade/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/home/trade/anaconda3/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/trade/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 535, in async_execute
    await self.async_execute_cell(
  File "/home/trade/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/trade/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
iris = sns.load_dataset("iris")    # 붓꽃 데이터
titanic = sns.load_dataset("titanic")    # 타이타닉호 데이터
tips = sns.load_dataset("tips")    # 팁 데이터
flights = sns.load_dataset("flights")    # 여객운송 데이터
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-1-d1c5953d5b0d>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0miris[0m [0;34m=[0m [0msns[0m[0;34m.[0m[0mload_dataset[0m[0;34m([0m[0;34m"iris"[0m[0;34m)[0m    [0;31m# 붓꽃 데이터[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0mtitanic[0m [0;34m=[0m [0msns[0m[0;34m.[0m[0mload_dataset[0m[0;34m([0m[0;34m"titanic"[0m[0;34m)[0m    [0;31m# 타이타닉호 데이터[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0mtips[0m [0;34m=[0m [0msns[0m[0;34m.[0m[0mload_dataset[0m[0;34m([0m[0;34m"tips"[0m[0;34m)[0m    [0;31m# 팁 데이터[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0mflights[0m [0;34m=[0m [0msns[0m[0;34m.[0m[0mload_dataset[0m[0;34m([0m[0;34m"flights"[0m[0;34m)[0m    [0;31m# 여객운송 데이터[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'sns' is not defined
NameError: name 'sns' is not defined

